python/ray/experimental/data/tests/test_dataset.py

```diff
@@ -104,6 +103,27 @@ def test_pyarrow(ray_start_regular_shared):
         .take() == [{"b": 2}, {"b": 20}]
 
 
+def test_uri_parser():
+    from ray.experimental.data.read_api import _parse_paths
+    fs, path = _parse_paths("/local/path")
```

Comment:
Will this work in CI (do we need to add s3fs as a dependency)?

Reply:
I checked the CI and test_uri_parser passed. But it failed at read_parquet and the error looks wired, like no PickleBuffer. Do you have some clue about it?